static/js/zamboni/users.js

```diff
@@ -3,6 +3,12 @@ $('#admin-login').click(function () {
   window.location = $(this).attr('data-url');
 });
 
+$('#fake_fxa_authorization .accounts a').click(function (e) {
+  $('#id_email').prop('value', $(this).prop('href').replace('mailto:', ''));
```

Comment:
I was wondering how a `mailto` would submit a form... Maybe it should not be `mailto` but a data attribute?

Reply:
Maybe... it was already in the text and I prefer having actual links but maybe I don't need to...

Reply:
(it doesn't matter much and I dont have a better idea right now fwiw)